Add @emoji support.

rjchee opened the pull request:
Does @emoji summon whoever emoji is?

rjchee committed:
add emoji support

rjchee committed:
nits

rjchee committed:
updated documentation

rjchee committed:
example image of @pokemon use

rjchee committed:
Update pokemon usage.

rjchee commented:
Also added documentation and stuff.

renxinhe commented on .gitignore:
```diff
@@ -17,4 +17,4 @@ temp_pokemon.png
 # Secret files
 secret.txt
-
+*.swp
```
What is this for?

renxinhe commented:
@rjchee can you also bump the version to `0.2.4` in `package.json`, and add `0.2.4: Added set emoji functionality.` to `README.md`?

rjchee commented:
Lol I added it because I wanted to ignore my vim swap files. Let me know if
you want that removed. Will bump version when I go home.

On Sat, Jul 30, 2016, 6:15 PM Jim Ren notifications@github.com wrote:

> @rjchee https://github.com/rjchee can you also bump the version to 0.2.4
> in package.json, and add 0.2.4: Added set emoji functionality. to
> README.md?
> 
> —
> You are receiving this because you were mentioned.
> 
> Reply to this email directly, view it on GitHub
> https://github.com/renxinhe/facebook-chat-bot/pull/15#issuecomment-236400926,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AImHaSF02oRNZ_H6jP2Hy9MAOcsmQVjGks5qa_cigaJpZM4JYx87
> .

rjchee committed:
updated version number, fixed typo

rjchee committed:
Merge branch 'master' of https://github.com/rjchee/facebook-chat-bot
